api/TimeAddressableMediaStore.yaml

```
@@ -1154,7 +1154,7 @@ paths:
         required: true
       responses:
         "204":
-          description: No content. The read_only property has been updated.
+          description: No content. The read_only property has been set.
```

Nit: Avoids a confusion with "set" being the opposite of "unset" rather than the common usage of "has taken on the given value"
```suggestion
          description: No content. The read_only property has been set to the given value.
```